doc-surrealdb_versioned_docs/version-1.x/surrealql/functions/geo.mdx

```diff
@@ -44,26 +44,35 @@ These functions can be used when working with and analysing geospatial data.
   </tbody>
 </table>
 
+## Point and geometry
+
+* A `point` is composed of two floats that represent the longitude (east/west) and latitude (north/south) of a location.
```

Comment:
Can we invert the order so that `latitude` comes before `longitude` avoid potential comprehension problem?

Reply:
This was a surprise to me as well but looks like GeoJSON specifies the other way around: https://en.wikipedia.org/wiki/GeoJSON#Geometries

Reply:
Ah, yes indeed. You are right.